bulk/node_bulk_load-AssetRecord.csv

```diff
@@ -1,167 +1,168 @@
+CGVEH-CWFPCP-30002,,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,MMP-7,ML14300-02,,20170627,,,Woods Hole Oceanographic Institution
+CGVEH-CWFPCP-71401,A00025,Node,0,Coastal Wire-Following Profiler,WFP COASTAL REFURBISHED,McLane,MMP-7,ML12714-01,,20110617,91556,,Woods Hole Oceanographic Institution
+CGVEH-CWFPCP-93601,A00391,Node,0,Coastal Wire-Following Profiler,WFP COASTAL (1ST ARTICLE),McLane,MMP-7,ML12936-01,,20130122,104442.23,,Woods Hole Oceanographic Institution
+CGVEH-CWFPCP-99101,A00287,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,MMP-7,ML12991-01,,20150713,92259.49,210-3510,Woods Hole Oceanographic Institution
+CGVEH-CWFPCP-99102,A00286,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,MMP-7,ML12991-02,,20130604,92259.49,210-3512,Woods Hole Oceanographic Institution
+CGVEH-CWFPCP-99105,A00304,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,MMP-7,ML12991-05,,20130510,92259.49,210-3512,Woods Hole Oceanographic Institution
+CGVEH-GLDRCE-00247,A00045,Node,1,Coastal Glider 247,COASTAL GLIDER 1000M,Teledyne Webb,CSG-1000,247,,20111007,201685,LOST AT SEA,Oregon State University
+CGVEH-GLDRCE-00311,A00078,Node,1,Coastal Glider 311,COASTAL GLIDER 200M,Teledyne Webb,CSG-0200,311,,20121002,186935,,Oregon State University
+CGVEH-GLDRCE-00312,A00079,Node,1,Coastal Glider 312,COASTAL GLIDER 1000M,Teledyne Webb,CSG-1000,312,,20121002,186935,,Oregon State University
+CGVEH-GLDRCE-00319,A00101,Node,1,Coastal Glider 319,COASTAL GLIDER 200M,Teledyne Webb,CSG-0200,319,,20121206,186935,,Oregon State University
+CGVEH-GLDRCE-00320,A00102,Node,1,Coastal Glider 320,COASTAL GLIDER 1000M,Teledyne Webb,CSG-1000,320,,20121206,186935,,Oregon State University
+CGVEH-GLDRCE-00326,A00158,Node,1,Coastal Glider 326,COASTAL GLIDER 200M,Teledyne Webb,CSG-0200,326,,20121228,186935,,Oregon State University
+CGVEH-GLDRCE-00327,A00159,Node,1,Coastal Glider 327,COASTAL GLIDER 200M,Teledyne Webb,CSG-0200,327,,20121228,186935,DESTROYED ON ROCKY SHORE,Oregon State University
+CGVEH-GLDRCE-00381,A00393,Node,1,Coastal Glider 381,COASTAL GLIDER 200M,Teledyne Webb,CSG-0200,381,,20130806,189935,,Oregon State University
+CGVEH-GLDRCE-00382,A00394,Node,1,Coastal Glider 382,COASTAL GLIDER 200M,Teledyne Webb,CSG-0200,382,,20130806,189935,,Oregon State University
+CGVEH-GLDRCE-00383,A00471,Node,1,Coastal Glider 383,COASTAL GLIDER 200M,Teledyne Webb,CSG-0200,383,,20130826,186935,,Oregon State University
+CGVEH-GLDRCE-00384,A00801,Node,1,Coastal Glider 384,COASTAL GLIDER 1000M,Teledyne Webb,CSG-1000,384,,20110628,186935,,Oregon State University
+CGVEH-GLDRCE-00386,A00473,Node,1,Coastal Glider 386,COASTAL GLIDER 1000M,Teledyne Webb,DG-1000,386,,20131011,186935,,Oregon State University
+CGVEH-GLDRCE-00871,,Node,1,Coastal Glider 871,COASTAL GLIDER 1000M,Teledyne Webb,GLD-0075_D,871,,20200813,,,Oregon State University
+CGVEH-GLDRCE-00917,,Node,1,Coastal Glider 917,COASTAL GLIDER,Teledyne Webb,GLD-0086_S,917,,20210403,,,Oregon State University

```

Comment:
I have removed the 200M because all of the Gliders (at least the EA Coastal glider specifically) are considered interchangeable now and what may have started as a 200M glider may be deployed next as a 1000M glider (even have 350 M pumps now).  The configuration is captured per deployment via the water depth in each deployment csv file.
I plan in a future update to AM to remove all references of 200M or 1000M so that it does not become ambiguous in metadata per deployment.

Reply:
Ah ok - sounds good.

Reply:
Looks fine.